src/Core/Services/IOrganizationService.cs

```diff
@@ -43,6 +43,7 @@ public interface IOrganizationService
     Task<List<Tuple<OrganizationUser, string>>> ConfirmUsersAsync(Guid organizationId, Dictionary<Guid, string> keys,
         Guid confirmingUserId, IUserService userService);
     Task SaveUserAsync(OrganizationUser user, Guid? savingUserId, IEnumerable<SelectionReadOnly> collections);
+    [Obsolete("EC-508 IDeleteOrganizationUserCommand should be used instead.")]
```

Comment:
We don't need to reference this Jira ticket in the comment here. It should reference a follow-up ticket that would complete this refactor and remove this endpoint, e.g. `IDeleteOrganizationUserCommand should be used instead. To be refactored and removed by EC-123`.

I don't think we have any tickets tracking the CQRS refactor beyond SCIM, please check with @MGibson1 how we're tracking this future work and consider creating a ticket for completing the removal of the `OrganizationService.DeleteUserAsync` method.

Reply:
Updated the comment with reference to the ticket EC-607 to address this in the future.